src/app/qgisapp.cpp

```diff
@@ -4436,6 +4451,22 @@ void QgisApp::askUserForOGRSublayers( QgsVectorLayer *layer )
   }
 }
 
+void QgisApp::addGeonodeLayer()
+{
+  QgsDebugMsg( "about to addGeonodeLayer" );
```

Comment:
Can you remove that (or if you think it's useful in the future, use `QgsDebugMsgLevel`)?